flags.html

```diff
@@ -4,12 +4,14 @@
         <link rel="stylesheet" href="flags.css">
     </head>
     <body>
-<div id="indonesia" class="flagContainer">
-    <div class="indonesiaRed"></div>
-</div>
-<!--<div id="columbia" class="flagContainer">-->
-
-<!--</div>-->
+<!--        <div id="indonesia" class="flagContainer">-->
+<!--            <div class="indonesiaRed"></div>-->
+<!--        </div>-->
+        <div id="columbia" class="flagContainer">
```

Comment:
this is certainly one way to do it, but there is a more succinct way, class wise:

```
<div id="columbia" class="flagContainer">
   <div class="halfPanel yellow">
   <div class="halfPanel">
       <div class="halfPanel blue"></div>
       <div class="halfPanel red"></div>
   </div>
</div>
```

Reply:
with halfPanel being simply
```
.halfPanel{
    height: 50%;
}
```

Reply:
with of course the various colors being set to their appropriate colors